tests/test_behavior/test_video_interface.py

```diff
@@ -239,7 +284,7 @@ def test_video_stub(self):
             nwbfile = io.read()
             mod = nwbfile.acquisition
             metadata = self.nwb_converter.get_metadata()
-            for video_index in range(len(metadata["Behavior"]["Videos"])):
-                video_interface_name = metadata["Behavior"]["Videos"][video_index]["name"]
+            for video_index in range(len(metadata["Behavior"]["Vide1"])):
```

Comment:
```suggestion
            for video_index in range(len(metadata["Behavior"]["Video1"])):
```
typo?

Reply:
Thank you @luiztauffer for working on this; I recently noticed this will be also useful for the Howe conversion! 💯